app/views/course/discussion/topics/_programming_file_annotation.html.slim

```diff
@@ -0,0 +1 @@
+// TODO: Implement displaying of code annotations
```

Comment:
This shouldn't be the convention. Each of the derived types should implement its own display.